pkg/gline/app.go

```diff
@@ -312,6 +342,26 @@ func (m appModel) attemptExplanation(msg attemptExplanationMsg) (tea.Model, tea.
 	})
 }
 
+func (m appModel) handleClearScreen() (tea.Model, tea.Cmd) {
+	// Try to execute /usr/bin/clear first
+	clearCmd := exec.Command("/usr/bin/clear")
+	clearCmd.Stdout = os.Stdout
+	clearCmd.Stderr = os.Stderr
+
+	err := clearCmd.Run()
+	if err != nil {
+		// If /usr/bin/clear fails, fall back to ANSI escape sequences
+		fmt.Print(CLEAR_SCREEN)
```

Comment:
curious why not always do this? what's the benefit of /usr/bin/clear?

Reply:
I was thinking that /usr/bin/clear is probably better at handling different edge cases when there is something different about the tty or the terminal emulator or whatever.  

But I’m fine if you’d prefer to omit that, let me know and I’ll patch it.

Reply:
yeah I think there's compatibility challenges with relying on /usr/bin/clear. For example we have a windows build as well.